.env.development

```diff
@@ -0,0 +1,3 @@
+REACT_APP_API_ROOT=https://api.nytimes.com/svc/topstories/v2/science.json
+# REACT_APP_API_QUERY_STRING
+REACT_APP_API_KEY=Gwxln5M3geWlhR6UE0TY1FUWKSG3wCil
```

Comment:
Despite accidentally committing the API key, it's impressive that you thought to include an `.env` file to protect the valuables. Very professional.